inferred-spans/build.gradle.kts

```diff
@@ -14,6 +14,7 @@ dependencies {
   implementation(libs.lmax.disruptor)
   implementation(libs.jctools)
   implementation(project(":common"))
+  implementation(libs.bundles.semconv)
```

Comment:
[for reviewer] this will add dependencies to both the stable and incubating artifacts.